pushmsixscript/src/pushmsixscript/microsoft_store.py

```diff
@@ -157,8 +159,30 @@ def _update_submission(config, channel, session, submission_request, headers, fi
     # update the in-progress submission, including uploading the new msix files
     application_id = config["application_ids"][channel]
     submission_id = submission_request.get("id")
+    submission_request, upload_file_names = _craft_new_submission_request_and_upload_file_names(config, channel, submission_request, file_paths, publish_mode)
     upload_url = submission_request.get("fileUploadUrl")
     upload_url = upload_url.replace("+", "%2B")
+
+    url = _store_url(config, f"{application_id}/submissions/{submission_id}")
+    response = session.put(url, _encode_submission_request(submission_request, encoding), headers=headers)
+    _log_response(response)
+    response.raise_for_status()
+    # Wrap all the msix files in a zip file and upload the zip
+    with tempfile.TemporaryDirectory() as work_dir:
+        zip_file_name = os.path.join(work_dir, "pushmsix.zip")
+        with zipfile.ZipFile(zip_file_name, "w") as zf:
+            for file_path in file_paths:
+                zf.write(file_path, arcname=upload_file_names[file_path])
+        # Note that simple HTTP uploads fail for large files (like ours!), so
+        # using the BlobClient is required.
+        blob_client = BlobClient.from_blob_url(upload_url)
+        with open(zip_file_name, "rb") as f:
+            d = blob_client.upload_blob(f, blob_type="BlockBlob", logging_enable=False)
+            log.debug(f"upload response: {d}")
```

Comment:
I didn't change this block. git believes I added these lines because `_craft_new_submission_request_and_upload_file_names()` reuses many lines above. It turns out it's more lines than this block alone. That's why git made this decision.